ocs_ci/ocs/resources/pod.py

```diff
@@ -163,7 +163,7 @@ def exec_bash_cmd_on_pod(self, command):
         Returns:
             str: stdout of the command
         """
-        cmd = f'exec {self.name} -- bash -c "{command}"'
+        cmd = f'exec {self.name} -- {sh} -c "{command}"'
```

Comment:
@petr-balogh - I see the original code is yours, could you please review this specific refactor?

Reply:
@petr-balogh please help check

Reply:
LGTM!